src/pages/DonationPage.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="custom-container">
+    <div class="static-page-wrapper">
+      <h1>{{ $t("donation.title") }}</h1>
+      <div>
+        <stripe-checkout
+          ref="checkoutRef"
+          mode="subscription"
+          :pk="publishableKey"
+          :line-items="lineItems"
+          :success-url="successURL"
+          :cancel-url="cancelURL"
+          @loading="v => loading = v"
+        />
+        <button @click="submit">{{ $t("donation.buttonSubscribe") }}</button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { StripeCheckout } from '@vue-stripe/vue-stripe';
+
+export default {
+  name: 'DonationPage',
+  components: {
+      StripeCheckout
+  },
+  data () {
+    this.publishableKey = 'pk_test_51I6cVwEUtJJGJFIUgsLpXxbk4DHr2g8Q2ZCIUlzSWbVBtWbcEZJdJuRhyjCxO0xFZdBecREiHx2zirBDPFN1qoeQ00A5Wd3bak' // process.env.STRIPE_PUBLISHABLE_KEY;
```

Comment:
Tole je potrebno premakniti v `.env` konfiguracije in nastaviti produkcijski key ko bomo srečni z implementacijo